Gordon360/Models/ViewModels/MailboxViewModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class MailboxViewModel
+    {
+        public string Combination { get; set;  }
+
+        public static implicit operator MailboxViewModel(Mailboxes req)
+        {
+            MailboxViewModel vm = new MailboxViewModel
+            {
+                Combination = (req != null) ? req.Combination ?? "" : "",
```

Comment:
I would like to keep it.

Reply:
Are you saying you would like to keep the original syntax? It's an amalgam of three different null checks and I don't think it works the way you think it does, although it does indeed have the same result as what Cam suggested.

Reply:
Ah, I see. That does make sense, and I understand the original intent of the code better now. That said, I do think Cameron's suggestion is more clear and idiomatic. The null-conditional operator is a standard feature of both C# and JavaScript. Its intent is exactly to access the `Combination` property if `req` is not null, or return null otherwise.

Reply:
Okay, will add.